apps/css-workshop/src/pages/alert.astro

```diff
@@ -46,15 +47,9 @@ import Layout from './_layout.astro';
           data-iui-status='informational'>Learn more</a
         >
       </span>
-      <button
-        aria-label='Close'
-        type='button'
-        class='iui-button'
-        data-iui-variant='borderless'
-        data-iui-size='small'
-      >
+      <Button_ aria-label='Close' type='button' variant='borderless' size='small'>
```

Comment:
`type="button"` should be set by default within the Button component, so that it doesn't need to be repeated for every instance.

Reply:
Will do.